- Description: cat /sys/bus/ccwgroup/drivers/lcs/0.0.ca00/type shows
               misleading device type
  Symptom:     the result of the command shows "OSA2 card" although
               it is an OSA Express card configured in LCS mode
  Problem:     There is no indication for cu3088 module to distinguish
               whether it is an OSA Express card configured in LCS mode
               or just an OSA 2 card, therefore the text for device type
               is hardcoded in cu3088 module
  Solution:    changed the device type text to "OSA LCS card"
  Problem-ID:  -
  Archs:       s390-31, s390-64

--- linux-2.5/drivers/s390/net/cu3088.c	14 Oct 2003 12:10:19 -0000	1.33
+++ linux-2.5/drivers/s390/net/cu3088.c	16 Jun 2004 14:35:38 -0000	1.33.12.1
@@ -1,5 +1,5 @@
 /*
- * $Id: cu3088.c,v 1.33 2003/10/14 12:10:19 cohuck Exp $
+ * $Id: cu3088.c,v 1.33.12.1 2004/06/16 14:35:38 pavlic Exp $
  *
  * CTC / LCS ccw_device driver
  *
@@ -38,7 +38,7 @@
 	"ESCON channel",
 	"FICON channel",
 	"P390 LCS card",
-	"OSA2 card",
+	"OSA LCS card",
 	"unknown channel type",
 	"unsupported channel type",
 };

